### ISSUE TITLE
Print group-members: KeyError encountered with cachememberinfo

### ISSUE THREAD

**Issue by onekmilesbehind** [taers232c/GAMADV-XTD3#445]
This could totally just be me messing up the syntax, but when attempting to leverage the cachememberinfo option in my gam print group-members command, it fails due to `KeyError: 'name'`. Re-running the command without cachememberinfo works fine. I'm trying to grab first and last name separately, which I believe triggers additional API calls, so I was hoping to use the cachememberinfo option to minimize these if possible. 

Here is an example of the full command:
```
gam print group-members select group1,group2,group3 cachememberinfo fields email userfields givenName,familyName todrive tdfileid **fileid** tdretaintitle tdsheet id:**sheetid** tdupdatesheet tdsheettitle "GAM Data" tdsheettimestamp tdsheettimeformat "Fetched %Y.%m.%dT%H_%M_%S" tdnobrowser tdnoemail
```

And the error I see (it exits after fetching members of one group):
```
Getting all Members, Managers, Owners for group1@domain.org (1/11)
Got 29 Members, Managers, Owners for group1@domain.org...
Traceback (most recent call last):
  File "__init__.py", line 76809, in ProcessGAMCommand
  File "__init__.py", line 33914, in doPrintGroupMembers
KeyError: 'name'
```

This is on GAMADV-XTD3 7.00.32. I've only been trying this for a week or so, so only tested against very recent versions.

-Rob

**Comment by taers232c** [comment]
I can duplicate, will fix.

Ross
----
Ross Scroggs
***@***.***

**Comment by taers232c** [comment]
Fixed 7.0.33
----
Ross Scroggs
***@***.***

**Comment by onekmilesbehind** [comment]
Thanks Ross! After updating to 7.0.33 I am able to use the cachememberinfo option, but am seeing two things now:

1. When I run the sample command I posted, it adds the 'name' field (so the full headers are group,email,name,name.familyName,name.givenName) whereas before it did not. The value isn't populated for most users (on a 700 line print, 101 users had 'name' populated). Is this expected when caching?

2. When I went to test the sample command without the cachememberinfo option, it yields a new error:
```
Getting all Members, Managers, Owners for group1@d181.org (1/11)
Got 29 Members, Managers, Owners for group1@d181.org...
Traceback (most recent call last):
  File "__init__.py", line 76810, in ProcessGAMCommand
  File "__init__.py", line 33909, in doPrintGroupMembers
KeyError: 'fullName'
```

-Rob
